zeroconf/__init__.py

```diff
@@ -2290,6 +2290,13 @@ def add_multicast_member(
         elif _errno in err_einval:
             log.info('Interface of %s does not support multicast, ' 'it is expected in WSL', interface)
             return None
+        elif _errno == 10042:
+            log.info(
+                'Failed to set socket option, this can happen if '
+                'the network adapter is in a disconnected state',
+                interface
```

Comment:
The `interface` parameter either needs to be removed or the format string needs to include it.